cfgov/v1/migrations/0006_update_tables.py

```diff
@@ -26,7 +26,7 @@ def update_content_tables(wagtail_page):
         if item['type'] == 'table':
             item['type'] = 'table_block'
             item['value'] = create_tableblock_data(old_table=item['value'])
-            print "Creating a new TableBlock on page %s" %(wagtail_page.title)
+            print "Creating a new TableBlock on page %s of page type %s" %(wagtail_page.title, wagtail_page.__class__.__name__)
 
 
 def update_full_width_text_tables(wagtail_page):
```

Comment:
@vccabral Ha, I wrote the original file as well so it can be in scope if we want it to be.  I did try to break out things into functions when it made semantic sense to, but I agree it's still too nested. Could you give an example of what you're thinking would resolve this?